### ISSUE TITLE
The ref keyword confuses the parser

### ISSUE THREAD

**Issue by mmano** [NaturalDocs/NaturalDocs#77]
The following prototype excludes the method from the documented stuff
```
static ref int RemoveBits<T>(this ref int value, T bits) where T:unmanaged, Enum
{ ../.. }
```

**Comment by NaturalDocs** [comment]
This was another easy one to fix.  This version should work, and has the other two fixes as well.

https://www.naturaldocs.org/download/temp/Natural_Docs_temp4.zip

**Comment by mmano** [comment]
It works ... Thx

**Comment by NaturalDocs** [comment]
This fix has been included in [Natural Docs 2.3 Development Release 1](https://www.naturaldocs.org/download/#Development).

**Comment by NaturalDocs** [comment]
Natural Docs 2.3 was released which includes this fix so I'm closing the issue.
